src/service/cron.js

```diff
@@ -3,10 +3,10 @@ var CronJob = require("cron").CronJob;
 
 export class CronController {
     cron(email) {
-        // new CronJob("*/15 * * * *", function() {
-            inbox.fetch_email(email); // running this function every 15 min
-            // inbox.Before_date_email(email, new Date());
-        // }, null, true);
+        new CronJob("*/15 * * * *", function() {
+            inbox.fetchEmail(email); // running this function every 15 min
+            inbox.beforeDateEmail(email, new Date());
```

Comment:
we are creating two function 1)fetchEmail fetch data from yesterday 
2)beforeDateEmail fetch data from before date